examples/pipelines/datacomp/components/mask_images/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN pip3 install --no-cache-dir -r requirements.txt
 
 # Install Fondant
 # This is split from other requirements to leverage caching
-ARG FONDANT_VERSION=main
+ARG FONDANT_VERSION=0.6.2
```

Comment:
Should we really change the Dockerfile? I think we can fix the version by using the fondant build command, right?

Reply:
right I just saw the build args command, I will revert it back